src/System.Net.Security/src/System/Net/Unix/CertModule.cs

```diff
@@ -20,7 +26,23 @@ internal override SslPolicyErrors VerifyCertificateProperties(X509Chain chain, X
                 sslPolicyErrors |= SslPolicyErrors.RemoteCertificateChainErrors;
```

Comment:
The Windows code returns if the chain fails AND the handle is invalid.  I feel like that if is "if Build returns true there'll be a handle; otherwise I guess we should check...".

Unix has no theoretical-equivalent state.  The handle is always representing null; and Build true means happy, Build false means validation errors.